generate-stackbrew-library.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 declare -A aliases
 aliases=(
-	[0.12.0]='0 latest'
+	[0.12.2]='0 latest'
```

Comment:
I've changed it manually for only rpi-node in PR for official-images repo. This issue will be fixed soon, So don't want to change it here.